src/index.html

```diff
@@ -8,6 +8,75 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <header>
+      <div class="stars">
+        <div class="stars--0">
```

Comment:
each container should have a block class (stars) and modifier class (stars--number)
```suggestion
        <div class="stars stars--0">
```